pyvista/core/pointset.py

```diff
@@ -27,6 +29,12 @@
 
 log = logging.getLogger(__name__)
 log.setLevel('CRITICAL')
+DEFAULT_INPLACE_WARNING = (
+    'You did not specify a value for `inplace` and the default value will '
+    'be changing to `False` in future versions for point-based meshes (e.g., '
+    '`PolyData`). Please make sure you are not assuming this to be an inplace '
+    'operation.'
+)
```

Comment:
I'm wondering if we should have this include the calling function's name so downstream users can more easily track down what exactly is throwing the error?